src/backend/main.js

```diff
@@ -1,44 +1,42 @@
 const redis = require('./redis')
-const p = require('bluebird')
+const bluebird = require('bluebird')
```

Comment:
w dużej aplikacji, gdzie cęsto byś tego używał, p pisze się fajniej niż bluebird. `p = require('bluebird')` to akurat dobra praktyka. Przy okazji jest mniej roboty ze zmianą biblioteki na inną w razie potrzeby